data-pipeline/src/data_pipeline/datasets/clinvar.py

```diff
@@ -105,8 +105,13 @@ def _parse_variant(variant_element):
     location_elements = variant_element.findall("./InterpretedRecord/SimpleAllele/Location/SequenceLocation")
     for element in location_elements:
         try:
+            chromosome = element.attrib["Chr"]
+            # A release in August 2022 introduced several Variants with a Chromosome of 'Un'
+            #   which caused failure of this pipeline when compared to the reference genome
+            if chromosome == "Un":
```

Comment:
Looks OK but I wonder if we want to add a warning to the effect of "Skipping Variant with bad chromosome value 'Un'", what do we do in comparable cases elsewhere in this codebase?

Reply:
I'm on board with adding a bit more output in special cases. In a similar case (resulting from a key error when parsing) simply skips the variant without output. It seems this "Un" is a bit more specific, and I've included output per your suggestion now.